routes/basic_db_functions_route.py

```diff
@@ -1,6 +1,5 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Form
```

Comment:
Consider removing unnecessary imports
```suggestion
from fastapi import APIRouter
```